modules/p2/ding_list/ding_list.install

```diff
@@ -37,6 +37,12 @@ function ding_list_install() {
   $mail_settings = variable_get('mail_system', array());
   $mail_settings['ding_list_share'] = "HTMLMailSystem";
   variable_set('mail_system', $mail_settings);
+
+  module_load_include('inc', 'page_manager', 'plugins/tasks/page');
+  $page = page_manager_page_load('user_bookmarks');
+  if (!empty($page)) {
+    page_manager_page_delete($page);
+  }
```

Comment:
If ding_list is enabled on a site, where the `user_bookmarks` page has been altered, that page is not removed when the feature no longer exists.

So when p2 is enabled on older sites the page might "survive". One could argue that if someone has customized the page, they might not want it deleted. So we can remove this code, I don't feel strongly about it.